Demo/Application/Features/PayPal - Native Checkout/BraintreeDemoPayPalNativeCheckoutViewController.swift

```diff
@@ -1,10 +1,28 @@
 import Foundation
 import UIKit
 import BraintreePayPalNativeCheckout
+import PayPalCheckout
```

Comment:
Agree with @jonathajones in previous PR that we do not want to require merchants to import `PayPalCheckout` to use specific features. Instead we should either create Braintree types that can be translated to the NXO types as or create a wrapper for the shipping callback types that can be passed through to the callback without the merchant having to pull in `PayPalCheckout` directly.

Reply:
Thanks for the feedback! That makes sense that we don't want merchants to import `PayPalCheckout` directly. I'll make the changes accordingly